openpype/hosts/maya/plugins/publish/collect_mayaascii.py

```diff
@@ -3,14 +3,14 @@
 import pyblish.api
 
 
-class CollectMayaAscii(pyblish.api.InstancePlugin):
+class CollectMayaScene(pyblish.api.InstancePlugin):
```

Comment:
This class name is already used for collecting the workfile itself and because of the clash publishing doesn't work. It is, however more of a problem of the other plugin where the name doesn' match the actual plugin python file name, so I reckon let's rename the class in `collect_scene.py` to `CollectWorkfile` and rename the same file to match other hosts `collect_workfile.py`

When that's done we can leave this class as is `COllectMayaScene` but let's please match the filename too. from `collect_mayaascii.py` to `collect_mayascene.py`

Reply:
Should it be `collect_maya_scene.py`?